common/genesis_wrapper.py

```diff
@@ -41,7 +41,11 @@ def define_wrapper(*args, **kwargs):
 
             outfile = run_genesis(self.__top_name, infiles, parameters,
                                   system_verilog=self.__system_verilog)
-            return m.DefineFromVerilogFile(outfile)[0]
+            return m.DefineFromVerilogFile(
+                outfile, type_map={"clk": m.In(m.Clock),
```

Comment:
Ideally `type_map` is a argument to this class.

Reply:
Done